### ISSUE TITLE
About training data

### ISSUE THREAD

**Issue by Flyige** [fuvvy666/GA-SVM#1]
i saw your code in main,but didn't see where is the matches_train_2.csv,consider the result_train is the lables of mathes_2.csv dataset ,to fit the features between training set and test set, in test111 and test222,each feature is 14 and 12, but i didn.t see the 12 features data in your trainning set, could you explain your dataset more sepcific or maybe you didn't put all the dataset? thx.
